<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(150, 150, 150);</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>571</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(202, 202, 195)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="Table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 211, 211)</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>981</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 35px; 
font-weight: bold;
color: white;
</string>
   </property>
   <property name="text">
    <string>Here are our preditctions for stop PLACEHOLDER STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_Button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>630</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; 

font-weight: bold;
color: white;
background-color:rgb(200, 200, 200)</string>
   </property>
   <property name="text">
    <string>Go back to route map</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
